docs/languages/en/tutorials/unittesting.rst

```diff
@@ -25,6 +25,25 @@ As the Zend Framework 2 API uses `PHPUnit <http://phpunit.de/>`_, so
 will this tutorial. This tutorial assumes that you already have PHPUnit
 installed. The version of PHPUnit used should be 3.7.*
 
+.. _setting-up-phpunit-to-use-composer-s-autoload-php:
+
+Setting up phpunit to use composer's autoload.php
+------------------------------
```

Comment:
Agreed, can you make this quick change @andy-b-84 then I'll get this merged - thanks for the PR.